invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/deposit/RDMDepositForm.js

```diff
@@ -115,379 +116,411 @@ export class RDMDepositForm extends Component {
+                    />
+                  </Overridable>
+                </AccordionField>
+              </Overridable>
+              <Overridable
+                id="InvenioAppRdm.Deposit.AccordionFieldBasicInformation.container"
+                config={this.config}
+                record={record}
+                vocabularies={this.vocabularies}
               >
-                {this.config.pids.map((pid) => (
-                  <Fragment key={pid.scheme}>
-                    <Overridable
-                      id="InvenioAppRdm.Deposit.PIDField.layout"
-                      pid={pid}
-                      record={record}
-                    >
-                      <PIDField
-                        btnLabelDiscardPID={pid.btn_label_discard_pid}
-                        btnLabelGetPID={pid.btn_label_get_pid}
-                        canBeManaged={pid.can_be_managed}
-                        canBeUnmanaged={pid.can_be_unmanaged}
-                        fieldPath={`pids.${pid.scheme}`}
-                        fieldLabel={pid.field_label}
-                        isEditingPublishedRecord={
-                          record.is_published === true // is_published is `null` at first upload
-                        }
-                        managedHelpText={pid.managed_help_text}
-                        pidLabel={pid.pid_label}
-                        pidPlaceholder={pid.pid_placeholder}
-                        pidType={pid.scheme}
-                        unmanagedHelpText={pid.unmanaged_help_text}
-                        required
-                      />
-                    </Overridable>
-                  </Fragment>
-                ))}
-
-                <Overridable
-                  id="InvenioAppRdm.Deposit.ResourceTypeField.layout"
-                  vocabularies={this.vocabularies}
+                <AccordionField
+                  includesPaths={[
+                    "metadata.resource_type",
+                    "metadata.title",
+                    "metadata.additional_titles",
+                    "metadata.publication_date",
+                    "metadata.creators",
+                    "metadata.description",
+                    "metadata.additional_descriptions",
+                    "metadata.rights",
+                  ]}
+                  active
+                  label={i18next.t("Basic information")}
                 >
-                  <ResourceTypeField
-                    options={this.vocabularies.metadata.resource_type}
-                    fieldPath="metadata.resource_type"
-                    required
-                  />
-                </Overridable>
+                  {this.config.pids.map((pid) => (
+                    <Fragment key={pid.scheme}>
+                      <Overridable
+                        id="InvenioAppRdm.Deposit.PIDField.container"
+                        pid={pid}
+                        record={record}
+                      >
```

Comment:
this in not ideal, overridable is created multiple times in a loop - with the same id (does it render when you test it?), I would rather extract it on the outside of the loop